utils/helpers/UiTransitionHelpers.ts

```diff
@@ -18,7 +18,7 @@ export const transitionFade = (
   enter: `ease-out transition duration-300 ${props?.enter || ""}`,
   enterFrom: `opacity-0 ${props?.enterFrom || ""}`,
   enterTo: `opacity-100 ${props?.enterTo || ""}`,
-  entered: `${props?.entered}`,
+  entered: `${props?.entered ||""}`,
```

Comment:
Shoot, this one too, it would say `undefined` as string in the rendered `class` prop